idaes/models_extra/column_models/tests/test_tray_column.py

```diff
@@ -139,7 +135,7 @@ def btx_fctp(self):
         # Inlet feed conditions
         m.fs.unit.feed.flow_mol_comp[0, "benzene"].fix(20)
         m.fs.unit.feed.flow_mol_comp[0, "toluene"].fix(20)
-        m.fs.unit.feed.temperature.fix(368)
+        m.fs.unit.feed.temperature.fix(368.001)
```

Comment:
I don't think this is a really good fix - it is just sidestepping the real issue.

Reply:
I'll work on the scaling

Reply:
Agree. Was this resolved?

Reply:
Yes, fixed with scaling